CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ This document describes changes between each past release.
 1.8.1 (unreleased)
 ------------------
 
+**Bug fixes**
```

Comment:
What do you mean ?

@ametaireau would be against this change ?

No problem, we can still discuss it, and reject it even if there's code attached...